### ISSUE TITLE
Laser Drill broken?

### ISSUE THREAD

**Issue by Leftorius** [powercrystals/MineFactoryReloaded#188]
here 2 screenshots, it doesnt work at all :(

http://i.imgur.com/MA2B5t8.jpg
http://i.imgur.com/U8nWAVR.png

it's the latest update, so :(

**Comment by powercrystals** [comment]
That's odd, and nobody else has reported this so I'm confused. Do the the prechargers have power? If you place and break a block between the prechargers and the laser does it activate? There might be railcraft heat nonsense in the way.

**Comment by Leftorius** [comment]
yes they have a full redstone energy cell next to it and the internal buffer has full energy. I replaced all of them but what do you mean with railcraft heat nonsense? :X

**Comment by powercrystals** [comment]
If there's an invisible non-air block between the precharger and the drill proper they won't be able to transmit power, and RC heat has a tendency to do that. That is weird though. Are you using a modpack and if so which one? What exact MFR version?

**Comment by Leftorius** [comment]
It's my privat server pack, so no ftb or similar. I'm using the latest release (Build 976). Tested in SSP.

**Comment by Technus** [comment]
It can be "RailCraft block hidden" set it's ID to 0 to disable that , also it ll make trackman's goggles less usefull

**Comment by Leftorius** [comment]
No I changed that to 0 and it's still not working... :/

**Comment by powercrystals** [comment]
I have no idea what to tell you then. Nobody else has reported this issue.

**Comment by Leftorius** [comment]
mh ;/

**Comment by jalapeno777** [comment]
I have been finding laser drills not operating unless very close to bedrock.  Even if there's only air between the drill and the bedrock if the laser is at around sea level it won't work.  I have to either move it to within a couple blocks of bedrock or spawn in a piece of bedrock close to the drill.
